src/main/webapp/app/exercises/shared/exercise-headers/header-exercise-page-with-details.component.html

```diff
@@ -130,6 +130,26 @@
                     }}
                 </span>
             </div>
+            <div *ngIf="exercise.presentationScoreEnabled">
+                <div *ngIf="(course?.presentationScore ?? 0) > 0">
+                    {{ 'artemisApp.courseOverview.exerciseDetails.presented' | artemisTranslate }}
+                    <span *ngIf="(studentParticipation?.presentationScore ?? 0) > 0" class="badge bg-success">
+                        {{ 'global.generic.yes' | artemisTranslate }}
+                    </span>
+                    <span *ngIf="(studentParticipation?.presentationScore ?? 0) <= 0" class="badge bg-secondary">
+                        {{ 'global.generic.no' | artemisTranslate }}
+                    </span>
+                </div>
+                <div *ngIf="!((course?.presentationScore ?? 0) > 0)">
```

Comment:
```suggestion
                <div *ngIf="!course?.presentationScore">
```

This could probably be even replaced with an angular template else